engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java

```diff
@@ -228,10 +246,10 @@ UnexpectedTypeException getNoValidatorFoundForTypeException(String constraintTyp
 	@Message(id = 63, value = "Unable to instantiate %s.")
 	ValidationException getUnableToInstantiateException(String className, @Cause Exception e);
 
-	ValidationException getUnableToInstantiateException(Class<?> clazz, @Cause Exception e);
```

Comment:
I confirm there's some magic involved: the second method reuses the message of the first method. So it's weird but OK.